src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -9778,6 +9778,70 @@ void br()
             _ => null
         };
     }
+}", changedOptionSet: changingOptions);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        [WorkItem(49725, "https://github.com/dotnet/roslyn/discussions/49725")]
```

Comment:
Is it okay for a work item to be a discussion instead of an issue?

Reply:
Yes, it's fine.